addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,425 @@
+        <item-type>Switch</item-type>
+        <label>Flash Briefing</label>
+        <description>Starts the flash briefing (Write Only)</description>
+    </channel-type>
+    <channel-type id="playWeatherReport">
+        <item-type>Switch</item-type>
+        <label>Weather Report</label>
+        <description>Starts the weather report (Write Only)</description>
+    </channel-type>
+    <channel-type id="playTrafficNews">
+        <item-type>Switch</item-type>
+        <label>Traffic News</label>
+        <description>Starts the traffic news (Write Only)</description>
+    </channel-type>
+    <channel-type id="playGoodMorning">
+        <item-type>Switch</item-type>
+        <label>Good Morning</label>
+        <description>Starts good moring news (Write Only)</description>
+    </channel-type>
+    <channel-type id="startRoutine">
+        <item-type>String</item-type>
+        <label>Start a routine</label>
+        <description>Type in what you normally say to Alexa without the preceding "Alexa," (Write Only)</description>
+    </channel-type>
+    <channel-type id="playAlarmSound">
+        <item-type>String</item-type>
+        <label>Alarm Sound</label>
+        <description>Plays an alarm sound</description>
+    </channel-type>
+    <channel-type id="amazonMusicTrackId">
+        <item-type>String</item-type>
+        <label>Amazon music track id</label>
+        <description>Id of the amazon music track</description>
+    </channel-type>
+    <channel-type id="amazonMusic">
+        <item-type>Switch</item-type>
+        <label>Amazon music</label>
+        <description>Amazon Music turned on</description>
+    </channel-type>
+    <channel-type id="amazonMusicPlayListId">
+        <item-type>String</item-type>
+        <label>Amazon music play list id</label>
+        <description>Amazon Music play list id (Write only, no current state)</description>
+    </channel-type>
+    <channel-type id="amazonMusicPlayListIdLastUsed">
+        <item-type>String</item-type>
+        <label>Amazon music last selected playlist id</label>
+        <description>Id of the playlist which was started with openHAB</description>
+    </channel-type>
+    <channel-type id="providerDisplayName">
+        <item-type>String</item-type>
+        <label>Provider Name</label>
+        <description>Name of music provider</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="bluetoothId">
+        <item-type>String</item-type>
+        <label>Bluetooth connection</label>
+        <description>MAC-Address of the bluetooth connected device</description>
+    </channel-type>
+    <channel-type id="bluetoothIdSelection">
+        <item-type>String</item-type>
+        <label>Bluetooth connection selection</label>
+        <description>Bluetooth connection selection (Currently only in PaperUI)</description>
+    </channel-type>
+    <channel-type id="imageUrl">
+        <item-type>String</item-type>

```

Comment:
No, because I would have to download every image of every song which was played on any echo. But only very view of them are really used, but the user opens the UI. But it would be nice to have an attribute for it in the xml to mark it as image url, so that the paper UI can display it as image in the same way as it is done in the sitemap.